docs/sorting-algorithms/selection-sort.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Selection Sort
+slug: /sorting-algorithms/selection-sort
+---
+
+# Introduction
+
+Selection Sort is a sorting algorithm which finds out the smallest element in unsorted part in each iteration and put it to the last of sorted part.
+It does less memory writes as compared to other sorting algorithms.
+
```

Comment:
Can you add an explanation for the algorithm?

Reply:
done